pkg/volume/util/hostutil/hostutil_linux.go

```diff
@@ -148,7 +148,11 @@ func (hu *HostUtil) GetFileType(pathname string) (FileType, error) {
 // PathExists tests if the given path already exists
 // Error is returned on any other error than "file not found".
 func (hu *HostUtil) PathExists(pathname string) (bool, error) {
```

Comment:
is this function periodically called? Just to make sure no log flood.

Reply:
Thanks for reminding, refer to other modules calling the `PathExists` method, all handle exceptions externally, and then output